lib/oneboxer.rb

```diff
@@ -407,7 +407,8 @@ def self.external_onebox(url, available_strategies = nil)
         ignore_hostnames: blocked_domains,
         force_get_hosts: force_get_hosts,
         force_custom_user_agent_hosts: force_custom_user_agent_hosts,
-        preserve_fragment_url_hosts: preserve_fragment_url_hosts
+        preserve_fragment_url_hosts: preserve_fragment_url_hosts,
+        timeout: 5
```

Comment:
We were using default FD timeout, which is 20 seconds and might be slightly too much?